packages/indexer-service/src/commands/start.ts

```diff
@@ -385,9 +406,7 @@ export default {
           allocationAmount: BigNumber.from('0'),
         },
       },
-      features: {
-        injectDai: true,
-      },
+      multiNetworks: undefined,
```

Comment:
do you see situations where multiple network is provided to the indexer service, such as multiple gateway endpoints?

Reply:
Absolutely! 
For now, users need to run one `indexer-service` instance per network. However, I believe a single instance could potentially handle multiple networks.